### PR TITLE
Add new /certified/desktops page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Add new /certified/desktop page

## QA
This is just a copy of the /certified code removing all category filtering

Check https://ubuntu-com-10108.demos.haus/certified/desktops

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4285

### CODE REVIEW

File: templates/certified/desktops-search-results.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+{% block title %}Certified hardware{% endblock %}
+
+{% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
+
+{% block content %}
+
+<form name="searchResults">
+ <section class="p-strip--suru-topped">
+   <div class="u-fixed-width">
+     <h1 class="u-sv3">Ubuntu certified
```
Comment (bethcollins92) [inline templates/certified/desktops-search-results.html:12]: Could you please remove the logic here and just have Ubuntu certified desktops for now? I'll be updating the top section of this page but for now it makes sense to have desktops in the title if you're at `/certified/desktops`

**Comment by bethcollins92** [comment]
Also, while you're there do you think you could update this "See all" button from the desktops card to point to `/certified/desktops` please? 

Screenshot 2021-08-02 at 11 43 37

File: templates/certified/desktops-search-results.html (hunk 1 above)
Comment (carkod) [reply templates/certified/desktops-search-results.html]: Good point. Done

File: templates/certified/desktops-search-results.html [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+{% block title %}Certified hardware{% endblock %}
+
+{% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
+
+{% block content %}
+
+<form name="searchResults">
+ <section class="p-strip--suru-topped">
+   <div class="u-fixed-width">
+     <h1 class="u-sv3">Desktops</h1>
```
Comment (bethcollins92) [inline templates/certified/desktops-search-results.html:12]: Sorry I meant "Ubuntu certified desktops"

Comment (bethcollins92) [reply templates/certified/desktops-search-results.html:12]:
```suggestion
     <h1 class="u-sv3">Ubuntu certified desktops</h1>
```

File: templates/certified/desktops-search-results.html [hunk 3]
```diff
@@ -0,0 +1,303 @@
+{% extends "certified/base_certification.html" %}
+
+{% block title %}Certified hardware{% endblock %}
+
+{% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
```
Comment (bethcollins92) [inline templates/certified/desktops-search-results.html:5]:
```suggestion
{% block meta_description %}Desktops that have been certified for use with Ubuntu.{% endblock meta_description %}
```

File: templates/certified/desktops-search-results.html (hunk 2 above)
Comment (carkod) [reply templates/certified/desktops-search-results.html]: Done

File: templates/certified/desktops-search-results.html (hunk 3 above)
Comment (carkod) [reply templates/certified/desktops-search-results.html]: Done

**Review by bethcollins92 — APPROVED** [review]
Thanks! LGTM

**Pull request merged**